src/MiNET/MiNET/Net/MCPE Protocol Documentation.md

```diff
@@ -2373,7 +2372,7 @@ Wiki: [Network Stack Latency Packet](https://github.com/NiclasOlofsson/MiNET/wik
 | Name | Type | Size |
 |:-----|:-----|:-----|
 |Timestamp | ulong |  |
-|IsFromServer | bool |  |
+|Unknown Flag | bool |  |
```

Comment:
In PM I call this `needResponse`. It's set to true in the ping and false in the ping.